custom-vu/src/main/kotlin/jces1209/vu/page/AbstractIssuePage.kt

```diff
@@ -5,4 +5,7 @@ interface AbstractIssuePage {
     fun waitForSummary(): AbstractIssuePage
     fun comment(): Commenting
     fun editDescription(description: String): AbstractIssuePage
+    fun linkIssueButton(): AbstractIssuePage
+    fun linkIssueSearchAndChoose(issuePrefix: String): AbstractIssuePage
+    fun linkIssueSubmit(): AbstractIssuePage
```

Comment:
Sub-features like commenting and linking can be encapsulated. Note that commenting contains of multiple steps, but it's concisely packaged as the `Commenting` interface. We can do the same with `Linking`.

- [ ] consider extracting `Linking`